ec3730/Models/Device/Sections/DataUsageInfoModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  DataUsageInfoModel.swift
+//  ec3730
+//
+//  Created by Ahmad Azam on 21/08/2022.
+//  Copyright © 2022 Zachary Gorak. All rights reserved.
+//
+
+import SwiftUI
+
+class DataUsageInfoModel: DeviceInfoSectionModel {
+    override init() {
+        super.init()
+        title = "Data Usage"
+
+        Task { @MainActor in
+            reload()
+        }
+    }
+
+    @MainActor override func reload() {
+        enabled = true
+        SystemDataUsage.reload()
+        rows.removeAll()
+        rows.append(CopyCellView(title: "Wifi", rows: [CopyCellRow(title: "Send / Recieve", content: SystemDataUsage.wifiTotal)]))
```

Comment:
3 rows? One for send, one receive, one total

Reply:
agree